src/InformationProcessing.go

```diff
@@ -131,3 +156,15 @@ func validateSkiFormat(fifaString string) bool {
 	tokens := strings.Split(firstLine, skiDelim)
 	return len(tokens) >= fifaTokenCount
 }
+
+func getScriptPath(opts *Opts) string {
+	sql := strings.HasSuffix(opts.ScriptName, ".sql")
```

Comment:
Make the check more reliable/case insensitive. Only one check is needed.

sql := strings.hasSuffix(strings.ToLower(opts.ScriptName), ".sql")